### PR TITLE
exceptions: all tasks

### PR DESCRIPTION
**Pull request by ivanP91**

### CODE REVIEW

File: src/com/clouway/task3/Demo.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.clouway.task3;
+
+import java.util.List;
+
+/**
+ * Created by clouway on 05.05.16.
+ */
+public class Demo {
+  public static void main(String[] args) throws Exception {
+    ArrayOperation arrayOperation = new ArrayOperation(3);
```
Comment (mgenov) [inline src/com/clouway/task3/Demo.java]: addElement(23) is a single operation
remove() is a single operation
so class cannot be named ArrayOperation when it executes multiple operations.

Comment (ivanP91) [inline src/com/clouway/task3/Demo.java]: fixed

File: src/com/clouway/task3/Demo.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
+package com.clouway.task3;
+
+/**
+ * Created by clouway on 05.05.16.
+ */
+public class Demo {
+  public static void main(String[] args) throws Exception {
+    Array array = new Array(3);
```
Comment (mgenov) [inline src/com/clouway/task3/Demo.java]: I don't see how you handle cases when array is full and you are trying to add a new element and when array is empty and your are trying to remove element.

File: src/com/clouway/task2/Checker.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+public class Checker {
+  /**
+   * checking of value whether is within range
+   * @param low    the first range
+   * @param high   the second range
+   * @param number input value
+   * @throws OutsideOfIntervalException
+   */
+  public void checking(int low, int high, int number) throws OutsideOfIntervalException {
```
Comment (mgenov) [inline src/com/clouway/task2/Checker.java]: checking is not verb.

Comment (mgenov) [inline src/com/clouway/task2/Checker.java]: If I have to check numbers if they are in interval between 0 and 100, I have to write the following boilerplate: 

```java
Checker checker = new Checker()
checker.check(0, 100, 4);
checker.check(0, 100, 10);
checker.check(0, 100, 105);
checker.check(0, 100, 33);
```
so, as you can see, I have to duplicate 0,100 on each row that I write which doesn't sound good.

**Pull request merged**